<launch>

    <!-- <node name="segment_slam" pkg="active_slam" type="segment_slam_node.py" output="screen" respawn="false" >
        <rosparam param="K" subst_value="True">[320.0, 0.0, 320.0, 0.0, 320.0, 240.0, 0.0, 0.0, 1.0]</rosparam>
        <rosparam param="distorition_params" subst_value="True">[0.0, 0.0, 0.0, 0.0]</rosparam>
        <rosparam param="T_BC" subst_value="True">[1.0, 0.0, 0.0, 0.5, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0]</rosparam>
    </node> -->


    <node name="segment_slam" pkg="active_slam" type="SAM_DA_node.py" output="screen" respawn="false" >
        <rosparam param="K" subst_value="True">[320.0, 0.0, 320.0, 0.0, 320.0, 240.0, 0.0, 0.0, 1.0]</rosparam>
        <rosparam param="distorition_params" subst_value="True">[0.0, 0.0, 0.0, 0.0]</rosparam>
        <rosparam param="T_BC" subst_value="True">[1.0, 0.0, 0.0, 0.5, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0]</rosparam>
    </node>

    <node name="segment_slam" pkg="active_slam" type="rover_map_node.py" output="screen" respawn="false" >
        <rosparam param="K" subst_value="True">[320.0, 0.0, 320.0, 0.0, 320.0, 240.0, 0.0, 0.0, 1.0]</rosparam>
        <rosparam param="distorition_params" subst_value="True">[0.0, 0.0, 0.0, 0.0]</rosparam>
        <rosparam param="T_BC" subst_value="True">[1.0, 0.0, 0.0, 0.5, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0]</rosparam>
    </node>

    <node name="segment_slam" pkg="active_slam" type="gt_locations_node.py" output="screen" respawn="false" >
    </node>


    <node name="segment_slam" pkg="active_slam" type="yolo_node.py" output="screen" respawn="false" >
    </node>

</launch>
